project/RELEASE-CHECKLIST.md

```diff
@@ -24,22 +24,45 @@ git remote add $GITHUBUSER git@github.com:$GITHUBUSER/docker.git
 
 ### 1. Pull from master and create a release branch
 
+#### Major releases
+
 Note: Even for major releases, all of X, Y and Z in vX.Y.Z must be specified (e.g. v1.0.0).
 
 ```bash
 export VERSION=vX.Y.Z
 git fetch origin
-git branch -D release || true
-git checkout --track origin/release
+git checkout --track origin/master
+git checkout -b release/$VERSION
```

Comment:
Wouldn't this technically be more accurate as just `vX.Y` since it's going to be the basis for all `.Z` releases of this major/minor combination?  (ie, something like `${VERSION%.*}` so `v1.2.3` becomes just `v1.2`)